server/pub/queryMany.ts

```diff
@@ -67,6 +67,12 @@ const createJoins = (query: PubsQuery) => {
 		if (typeof hasReviews === 'boolean') {
 			builder.leftOuterJoin('ReviewNews', 'Pubs.id', 'ReviewNews.pubId');
 		}
+		if (submissionStatus) {
+			builder.innerJoin('Submissions', {
+				'Submissions.pubId': 'Pubs.id',
+				'Submissions.status': knex.raw('some(?::string[])', [submissionStatus]),
```

Comment:
should knex.raw be builder.where

Reply:
I don't think so. The object `{ 'Submissions.status': X }` is acting as the `ON` clause of a `JOIN` query (kind of like a `WHERE`)  and this one key-value pair renders down to `"Submissions"."status" = X`. The `X` here is the query builder carefully injecting our statuses array into the query and casting it to an array of strings with `::string[]`. The `some()` is checking to see if that equality is true of _any_ element in the array (it is a weird syntax).